### DISCUSSION TITLE
.net 8.0

### DISCUSSION THREAD

**Discussion by YamamotoKansuke** [Shane32/QRCoder#563] [General]
Given that .net 6.0 support is ending November this year, any changes for support for .net 8.0?

**Comment by codebude** [comment]
QRCoder already supports net8.0. It's compiled against net6.0 which reads as: Supports net6.0 and newer.

**Reply by YamamotoKansuke** [reply]
Although you would think so, the renderer QrCoder is not available in .net8.0, 
I've tried it, doesn't work.

For reference:
https://github.com/codebude/QRCoder/wiki/Advanced-usage---QR-Code-renderers#21-qrcode-renderer-in-detail
